requirements/base.txt

```diff
@@ -7,7 +7,7 @@ boto3~=1.9, >=1.9.56
 PyYAML~=3.12
 cookiecutter~=1.6.0
 aws-sam-translator==1.10.0
-docker>=3.3.0
+docker>=3.7.0
```

Comment:
If we are not planning to update to the latest, can we bump this (and pin) to the latest 3.x [3.7.2](https://github.com/docker/docker-py/releases/tag/3.7.2)

Reply:
Actually, we should stick to ~3.7.0 which will upgrade patch versions...

Reply:
That means build are not deterministic though.

Reply:
Yes, but patch versions shouldn't break..